j2lint/utils.py

```diff
@@ -25,17 +23,30 @@ def load_plugins(directory):
     """
     result = []
     file_handle = None
-
-    for pluginfile in glob.glob(os.path.join(directory, '[A-Za-z]*.py')):
-        pluginname = os.path.basename(pluginfile.replace('.py', ''))
+    for plugin_file in glob.glob(os.path.join(directory, "[A-Za-z_]*.py")):
+        plugin_name = os.path.basename(plugin_file.replace(".py", ""))
         try:
-            logger.debug("Loading plugin {}".format(pluginname))
-            spec = importlib.util.spec_from_file_location(
-                pluginname, pluginfile)
-            module = importlib.util.module_from_spec(spec)
-            spec.loader.exec_module(module)
-            obj = getattr(module, pluginname)()
```

Comment:
Hmm here the code was returning an instance of the Rule wherease we are now returning the Class - though through testing it seems everything is still working. Need to double check this

Reply:
ok this is going with another change in collection where we now instantiate the rule_obj there